bot/constants.py

```diff
@@ -45,13 +58,26 @@ class Client(NamedTuple):
     season_override = environ.get('SEASON_OVERRIDE')
 
 
+class Colours:
+    soft_red = 0xcd6d6d
+    soft_green = 0x68c290
+    dark_green = 0x1f8b4c
+    orange = 0xe67e22
+
+
+class Emojis:
+    star = "\u2B50"
+    christmas_tree = u"\U0001F384"
```

Comment:
Hm, do we need `u` prefixes on 3?

Reply:
No, I didn't write this though. I only moved the ordering of the constants classes as per lemon's suggestions. Since I'm here anyway, I'll remove it.